app/src/components/inputs/input/input.jsx

```diff
@@ -44,36 +45,42 @@ export const Input = ({
   onKeyPress,
   touched,
   asyncValidating,
+  inputBadge,
 }) => (
-  <input
-    ref={refFunction}
-    type={type}
-    className={cx('input', `type-${type}`, className, {
-      'mobile-disabled': mobileDisabled,
-      disabled,
-      error,
-      touched,
-      readonly,
-      asyncValidating,
-    })}
-    value={value}
-    placeholder={placeholder}
-    maxLength={maxLength}
-    disabled={disabled}
-    readOnly={readonly}
-    onChange={onChange}
-    onFocus={onFocus}
-    onBlur={onBlur}
-    onKeyUp={onKeyUp}
-    onKeyPress={onKeyPress}
-  />
+  <Fragment>
+    <input
+      ref={refFunction}
+      type={type}
+      className={cx('input', `type-${type}`, className, {
+        'mobile-disabled': mobileDisabled,
+        disabled,
+        error,
+        touched,
+        readonly,
+        asyncValidating,
+      })}
+      value={value}
+      placeholder={placeholder}
+      maxLength={maxLength}
+      minLength={minLength}
+      disabled={disabled}
+      readOnly={readonly}
+      onChange={onChange}
+      onFocus={onFocus}
+      onBlur={onBlur}
+      onKeyUp={onKeyUp}
+      onKeyPress={onKeyPress}
+    />
+    {inputBadge && <span className={cx('input-badge')}>{inputBadge}</span>}
```

Comment:
I think its better to do it in `InputControl` component. It always ease to add some caption near the input field.
Not needed to do it in component directly.
OR just use the `tip` prop from `ModalField` component - it requires less changes.